.drone.yml

```diff
@@ -856,7 +918,73 @@ steps:
 
 ---
 kind: pipeline
-name: docker-manifest
+name: docker-linux-arm64-release
+
+platform:
+  os: linux
+  arch: arm64
+
+depends_on:
+  - testing-amd64
+  - testing-arm64
+
+trigger:
+  ref:
+  - refs/heads/main
+  event:
+    exclude:
+    - cron
+
+steps:
+  - name: fetch-tags
```

Comment:
Does this need the `fetch-tags` step since we are not using version tags in the docker build step?

Not sure if the dockerfiles need tags for `RUN if [ -n "${GITEA_VERSION}" ]; then git checkout "${GITEA_VERSION}";`, but if I understand the logic, `if [ -n "${GITEA_VERSION}" ]` should always resolve false here for dev build.

Reply:
It does need the fetch-tags logic, as otherwise we can't generate a version based on git information (eg. `1.15.0+dev-582-g3dcb3e907`).